package.json

```diff
@@ -1,43 +1,54 @@
 {
-  "name": "jam-bot",
-  "version": "1.0.0",
-  "description": "A bot for the CowodingJam server",
-  "main": "dist/start.js",
-  "private": "true",
-  "scripts": {
-    "prettify": "eslint --fix",
-    "build": "tsc",
-    "run": "node dist/start.js",
-    "start": "npm run prettify && npm run build && npm run run"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Wonderful-bot/Jam-Bot.git"
-  },
-  "contributors": [
-    "Commentator2.0 aka Leonie <kommentator2.9.0.1@gmail.com>",
-    "Lucy <lucy@blackdemon.codes>",
-    "Antti <antti@antti.codes>",
-    "WilsontheWolf (https://wilson.antti.codes)",
-    "Murph - StrangeGirlMurph",
-    "Amber - UselessAmber",
-    "Floschy - flloschy",
-    "Riley - Exonorid",
-    "Crispr - CrispiCas",
-    "Kryal - Da-Real-Kryall",
-    "Ian - IanToujou"
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/Wonderful-bot/Jam-Bot/issues"
-  },
-  "homepage": "https://github.com/Wonderful-bot/Jam-Bot#readme",
-  "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.15.0",
-    "@typescript-eslint/parser": "^5.15.0",
-    "eslint": "^8.11.0"
-  },
-  "dependencies": {
-    "typescript": "^4.6.2"
-  }
+	"name": "jam-bot",
+	"version": "1.0.0",
+	"description": "A bot to manage a whole game jam server from a fully functional poll-system to managing color roles.",
+	"main": "dist/start.js",
+	"private": "true",
+	"type": "module",
+	"scripts": {
+		"prettify": "eslint ./src/ --fix && npx prettier -w .",
+		"build": "tsc -p tsconfig.json",
+		"run": "node --experimental-specifier-resolution=node .",
+		"start": "npm run prettify && npm run build && npm run run"
```

Comment:
Why do we run preffiter on start? First off depending on rules it might mess with incomplete stuff, plus its just delaying the run with code stuff